### DISCUSSION TITLE
What's new for the Visual Basic Application Framework

### DISCUSSION THREAD

**Discussion by melytc** [dotnet/project-system#8913] [Announcements]
We have invested into modernizing the experience for developers, focusing on enhancing productivity and a modern look and feel with the new Project Properties UI. VB users might have noticed the Application Framework has its own section now! 🥳

Head over to our [.NET blog post](https://devblogs.microsoft.com/dotnet/update-to-winforms-vb-appframework/) to read more about each property and learn what happens behind the scenes to generate the Application Framework properties' code. It's the right time to think about **modernizing your older .NET Framework VB apps and bring them over to .NET**: not only has now a modern UX, but it also includes new features and considerable performance improvements from the WinForms and .NET runtime! 

![image](https://user-images.githubusercontent.com/8518253/224197112-eff7af7a-8937-4111-8946-c96e38054cba.png)

As always, feedback is welcomed (feel free to comment here or open a new issue). ❤️

**Comment by paul1956** [comment]
Glad you got Minimum Splash Screen Display Time it was holding back 1 very old app, fantastic job, thanks all.

**Reply by melytc** [reply]
Sorry we had you waiting on it; let us know if you have any feedback 🙂 
You should be able to see the Splash Screen properties in 17.7 Preview 1 🎉
